lib/callback_model.js

```diff
@@ -28,8 +28,13 @@ class CallbackModel extends EventEmitter {
     this.connection._updateSecret(newSecret, reason, cb);
   }
 
-  createChannel (cb) {
+  createChannel (options, cb) {
+    if (arguments.length === 1) {
+      cb = options;
+      options = undefined;
+    }
     var ch = new Channel(this.connection);
+    ch.setOptions(options);
```

Comment:
I used a setter to specify the channel options to avoid making a breaking change to the Channel constructor